punishments.js

```diff
@@ -525,3 +612,17 @@ Punishments.checkIpBanned = function (connection) {
 
 	return banned;
 };
+
+Punishments.checkRoomBanned = function (user, roomid) {
+	if (!user) return;
+
+	if (Punishments.roombannedUsers.has(roomid + '|' + user.userid)) {
```

Comment:
This is not how to do the check, go read my other code for how to do the check.

1. my punishment code doesn't use timeouts, so it needs to check for expiration at every read

2. doing `.has` and then immediately doing `.get` is a bad pattern, refer to how I do it